tokio-quiche-utils/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[workspace]
+members = ["./crates/*"]
+resolver = "2"
+
+[workspace.package]
+version = "0.1.0"
```

Comment:
For other crates in the repo we don't use a single version, to avoid having to publish all crates every time we make a release. So, IMo each crate should have its own version so they can be released independently.